torch/_inductor/codegen/triton_combo_kernel.py

```diff
@@ -832,7 +838,15 @@ def codegen_kernel(self, name: Optional[str] = None) -> str:
 
         argdefs, _, signature, _ = self.args.python_argdefs()
         argdefs = self.add_numel_to_args(argdefs, signature)
-        argdefs = self.add_blockd_to_args(argdefs)
+        block_args = self.get_block_args()
+        if self.enable_autotune:
+            argdefs.extend([f"{x.name}: tl.constexpr" for x in block_args])
+            if (
+                get_triton_attrs_descriptor_version()
+                == TritonAttrsDescriptorVersion.V4_DICT
```

Comment:
(Throughout the PR) I worry what happens when Triton introduces `V5_DICT`-- with this framing we will default this code to off in newer versions -- while I think defaulting to on might be more robust.

Suggest creating a helper method for this comparison so we can only update it in one place.